sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -44,7 +44,7 @@
    "metadata": {},
```

Comment:
<p>Why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks for pointing this out. I see that we really don't need this s3fs since this is only used to download the Debugger reports. We can use the boto3 S3 client instead. I'll fix it in a follow up PR.</p><p> </p><p> </p>